src/core/dnd/types.ts

```diff
@@ -6,27 +6,42 @@
  * ---------------------------------------------------------------------
  */
 
+import { SchemaElement } from '../model';
 import { LinkedUISchemaElement } from '../model/uischema';
 
-export const UI_SCHEMA_ELEMENT: 'uiSchemaElement' = 'uiSchemaElement';
+export const NEW_UI_SCHEMA_ELEMENT: 'newUiSchemaElement' = 'newUiSchemaElement';
+export const MOVE_UI_SCHEMA_ELEMENT: 'moveUiSchemaElement' =
+  'moveUiSchemaElement';
 
-export type DndType = DragUISchemaElement;
+export type DndType = DragNewUISchemaElement | DragMoveUISchemaElement;
```

Comment:
The `DragNew...` `DragMove...` feels a little bit clunky. I think we can remove the `Drag` in general as we are within the `DndType`.